src/main/java/core/basesyntax/GenerateArray.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class GenerateArray {
+    private static Colors[] color = Colors.values();
+
+    public static void main(String[] args) {
+        Random rnd = new Random();
+        int x = rnd.nextInt(10);
+        Object [] randomArray = new Object[x];
+        for (int i = 0; i < x; i++) {
+            randomArray[i] = random();
+        }
+        for (Object print: randomArray) {
+            System.out.println(print.toString());
+        }
+    }
+
+    private static Object random() {
```

Comment:
let's move this method to the separate class FigureGenerator) and rename this class to the Application or ProgramRunner with one main() method

Reply:
done